automation_tools/repository.py

```diff
@@ -153,6 +153,7 @@ def enable_satellite_repos(cdn=False, beta=False, disable_enabled=True,
     ]
     if sat_version == '6.4':
         repos.append('rhel-{0}-server-ansible-2-rpms')
+        repos.append('rhel-7-server-satellite-maintenance-6-rpms')
```

Comment:
could be {0} instead of '7' 

if future then future, you considered sat 6.5,6.6... but not rhel8 :-)

Reply:
no it will be only 6 no matter what sat version is...currently it has support for all 6.1, 6.2, 6.3 and 6.4.